### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by Dnyandeo33** [lab-brussels-1/group6#19] [General]
Review

**Comment by sammou00** [comment]
Hello,

1- Very good job, we all very proud of you.

2- Nice `user-personas`, but it could be more than on user.

3-  Your `constrains`  are too much I think mentioning the most important constrains are enough 

see [group 3 ] (https://github.com/lab-brussels-1/All-About-Trees-Group-3/blob/master/planning/constrain.md)

4- Your `design` is great , just a small details, images normally represented by  
![image](https://user-images.githubusercontent.com/67711948/159986917-fe15a122-ba61-45ee-bdb3-1d0265fd08ef.png)


5-  Very nice `development-strategy` 

6- `Backlog` I think you should add more stories like images , footers 
![image](https://user-images.githubusercontent.com/67711948/159987164-1fd67db3-16c8-4602-ab40-3b3a83e1169e.png)

7- No comments in your `css` why ?

**Comment by bermarte** [comment]
- Personally I would remove the PRs from the Project Board, because otherwise your PB will be really stuffed.

- I see many tickets/issues without description (remember what we did during the class).

- The goal of this assignment was the workflow, not really working on the HTML and the CSS (this is for the next project).

**Comment by sammou00** [comment]
@bermarte  Yes, we can explain some important points on Sunday.
